isos/bootstrap.sh

```diff
@@ -82,6 +82,7 @@ fi
 
 # copy in our components
 cp ${BIN}/tether-linux $(rootfs_dir $PKGDIR)/bin/tether
+cp ${BIN}/unpack $(rootfs_dir $PKGDIR)/bin/unpack
```

Comment:
this is also unnecessary; I added it to bootstrap while chasing a bug and need to remove it